WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -49,7 +49,7 @@ final class OrderListViewModel {
         }
 
         /// Enabled if site is launched, has published at least 1 product and set up payments.
-        return site.isPublic && hasAnyPaymentGateways && hasAnyPublishedProducts
+        return (site.visibility == .publicSite) && hasAnyPaymentGateways && hasAnyPublishedProducts
```

Comment:
This is likely an error that existed previously since the WordPressSite had the `isPublic` property set to `false`, which was equivalent to `comingSoon`. Therefore, with the configuration I had in place, there should not have been any difference in behavior.